### DISCUSSION TITLE
怎么隐藏El-Table的滚动条(How to hide scrollbar in a table row)

### DISCUSSION THREAD

**Discussion by saddit** [element-plus/element-plus#5555] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/55338151/150628622-b58fa9d0-2c1a-44cb-bccd-0a98f6c54d34.png)
隐藏图片中的那个滚动条，在缩小窗口时出现，完全展开则没有

**Comment by msidolphin** [comment]
是否更新了最新版？如果是的话你能提供一个复现demo吗？

**Comment by msidolphin** [comment, marked as answer]
这个问题已经找到原因了
